Cubical/Data/Int/Properties.agda

```diff
@@ -300,6 +316,102 @@ private
                                        (minusPlus m)
                                        (plusMinus m))
 
++inv : ∀ z → z - z ≡ pos zero
```

Comment:
Maybe `-Cancel`?

Reply:
Makes sense. +inv would make more sense as part of a group constructor than it does here.